### DISCUSSION TITLE
Help, youtube is now throwing adblock detection on pc browsers!

### DISCUSSION THREAD

**Discussion by terrafirma2021** [TeamNewPipe/NewPipe#10492] [Questions]
Can anybody advise how newpipe can be adapted to run on a webserver, so we can use YouTube without ads?

**Comment by ChiefMikeK** [comment]
Please give your actual and detailed use case since i've not been getting ads in NewPipe or on several browsers🙄

also check sub-reddit  
- https://www.reddit.com/r/Adblock/comments/174fplr/youtubes_antiadblock_popup_solution_turns_out/

**Reply by terrafirma2021** [reply]
Thanks for the reply, but opus solved the issue.
The Detection of AdBlock is linked to the account, some get detection and some don't, it's not related to any add-on configuration,
It's part of the manifest V3 roll out and the terms of service changing for YouTube.
If you look further than the one Reddit post, you will see many people have this issue now, and the solution will be piped YouTube from now on out.


If you keep skipping the message, you will locked out of YouTube.

This question will serve as great help for anybody who is locked out of youtube 🥇 

![YKQYTBK](https://github.com/TeamNewPipe/NewPipe/assets/69117226/16a2943b-838a-42c8-a737-3679b15ebc9e)

**Reply by terrafirma2021** [reply]
https://blog.adblockplus.org/blog/whats-happening-with-youtube-ads#c008115

Its posted here for anybody who may want to look into this issue.

They do mention that it's account related and logging out solves the issue
However, the Piped YouTube will be the fix long term 

Thanks Newpipe team, <3

**Comment by opusforlife2** [comment]
https://piped.video/

**Reply by terrafirma2021** [reply]
Thank you, This is perfect and what is needed.
